azure-pipelines/testfx.yml

```diff
@@ -1,30 +1,40 @@
 steps:
+
+- task: UseDotNet@2
+  displayName: Install .NET Core SDK 2.2.300
+  inputs:
+    packageType: sdk
+    version: 2.2.300
+
+- task: UseDotNet@2
+  displayName: Install .NET Core 2.1 runtime
+  inputs:
+    packageType: runtime
+    version: 2.1.x
+
+- script: dotnet --info
+  displayName: Show dotnet SDK info
+
 - script: dotnet restore
   displayName: Restore packages
   workingDirectory: ${{ parameters.projectdirectory }}
-- script: dotnet build -v n -f netcoreapp1.0 -c $(BuildConfiguration) --no-restore /bl:"$(Build.ArtifactStagingDirectory)/build_logs/netcoreapp1.0.binlog"
-  displayName: Build test for netcoreapp1.0
-  workingDirectory: ${{ parameters.projectdirectory }}
-- script: dotnet build -v n -f netcoreapp2.0 -c $(BuildConfiguration) --no-restore /bl:"$(Build.ArtifactStagingDirectory)/build_logs/netcoreapp2.0.binlog"
-  displayName: Build test for netcoreapp2.0
-  workingDirectory: ${{ parameters.projectdirectory }}
 
-- script: dotnet test -v n -f netcoreapp1.0 -c $(BuildConfiguration) --no-build --filter "TestCategory!=FailsInCloudTest"
-  displayName: Run tests for netcoreapp1.0 on the .NET Core 1.0.11 runtime
+- script: dotnet build --no-restore -v n -c $(BuildConfiguration) /bl:"$(Build.ArtifactStagingDirectory)/build_logs/build.binlog"
+  displayName: Build test
   workingDirectory: ${{ parameters.projectdirectory }}
-  env:
-    DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX: 0
-    RuntimeFrameworkVersion: 1.0.11
-- script: dotnet test -v n -f netcoreapp1.0 -c $(BuildConfiguration) --no-build --filter "TestCategory!=FailsInCloudTest"
-  displayName: Run tests for netcoreapp1.0 on the .NET Core 1.1.8 runtime
+
+- script: dotnet test -v n -f netcoreapp2.1 -c $(BuildConfiguration) --no-build --filter "TestCategory!=FailsInCloudTest"
+  displayName: Run tests for netcoreapp2.1 on the .NET Core 2.1 runtime
   workingDirectory: ${{ parameters.projectdirectory }}
   env:
     DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX: 0
-    RuntimeFrameworkVersion: 1.1.8
-- script: dotnet test -v n -f netcoreapp2.0 -c $(BuildConfiguration) --no-build --filter "TestCategory!=FailsInCloudTest"
-  displayName: Run tests for netcoreapp2.0 on the .NET Core 2.0.7 runtime
+    RuntimeFrameworkVersion: 2.1
+  enabled: false # TODO: figure out why this step hangs
```

Comment:
Did you create an issue for this?

Reply:
This was a pre-existing issue. You'll see the line below as well (without red/green). 
I hadn't filed an issue to track this. But I just did: https://github.com/microsoft/vs-threading/issues/499